ui/app/components/app/permission-page-container/permission-page-container-content/permission-page-container-content.component.js

```diff
@@ -64,17 +63,13 @@ export default class PermissionPageContainerContent extends PureComponent {
 
   renderRequestedPermissions () {
     const {
-      selectedPermissions, permissionsDescriptions, onPermissionToggle,
+      selectedPermissions, onPermissionToggle,
     } = this.props
     const { t } = this.context
 
     const items = Object.keys(selectedPermissions).map((methodName) => {
 
-      // the request will almost certainly be reject by rpc-cap if this happens
```

Comment:
Despite the suggestion, this isn't a thing. rpc-cap does expect that those strings exist but doesn't actually check that or use them for anything—I've opened MetaMask/rpc-cap#99 to represent removing that.